caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditRestHighLevelClientFactory.java

```diff
@@ -50,6 +50,11 @@ private ElasticAuditRestHighLevelClientFactory() {
      */
     public static RestHighLevelClient getHighLevelClient(final String hostAndPortValues)
         throws AuditConfigurationException {
+        String elasticProtocol = System.getProperty(ElasticAuditConstants.ConfigEnvVar.CAF_ELASTIC_PROTOCOL,
+                System.getenv(ElasticAuditConstants.ConfigEnvVar.CAF_ELASTIC_PROTOCOL));
+        if (elasticProtocol == null) {
+            elasticProtocol = ElasticAuditConstants.ConfigDefault.CAF_ELASTIC_PROTOCOL;
+        }
```

Comment:
Shouldn't the configuration be read in the same place where all the other configuration is read?

Reply:
Initially, I added along with other environment variables section in ElasticAuditConnection. But that made changes to add an extra parameter to getHighLevelClient. Due to this extra parameter, there are quite a few test classes that need to be updated.  

For one of these variables, I don't want to touch many places in test classes. But now I moved it to ElasticAuditConnection class.